app/konnect/analytics/services-and-routes.md

```diff
@@ -3,9 +3,8 @@ title: Analyze Services and Routes
 content_type: how-to
 ---
 
-<!-- this page is deprecated-->
 
-In the Service Hub, the service, service version, and route graphs provide dynamic
+In the API product builder the API product, API product version, and route graphs provide dynamic
 graphs with up to 30 days of data. To view data beyond this time frame, export
 the data into a comma-separated values (CSV) file.
```

Comment:
We don't support extending the time range any more for the CSV export -- it's only possible to export the data in the chart.